Formula/haskell-language-server-wrapper.rb

```diff
@@ -0,0 +1,34 @@
+class HaskellLanguageServerWrapper < Formula
+  desc "HLS - wrapper binary"
+  homepage "https://github.com/haskell/haskell-language-server"
+  url "https://github.com/haskell/haskell-language-server/archive/1.0.0.tar.gz"
+  sha256 "14e28d6621d029f027fae44bc4a4ef62c869dab24ff01b88a2e51e6679cbff6c"
+  license "Apache-2.0"
+  head "https://github.com/haskell/haskell-language-server.git"
+
+  # we need :github_latest here because otherwise
+  # livecheck picks up spurious non-release tags
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "cabal-install" => [:build, :test]
+  depends_on "ghc" => [:build, :test]
+
+  def install
+    system "cabal", "v2-update"
+    system "cabal", "v2-install", *std_cabal_v2_args
+    rm bin/"haskell-language-server"
+  end
```

Comment:
It looks like the wrapper is built whatever you do. Why can't I just use the wrapper that gets build for ghc 8.8 when I want to use ghc 8.8?

Reply:
Or, alternatively, the wrapper that gets build in the existing `haskell-language-server` formula for ghc 8.8, etc.

Reply:
Both binaries are always built. @jneira mentioned that it might be a Cabal issue.
When I open PRs for HLS@ghc8.6 / 8.8 they will also require the wrapper. If users install more than one HLS@GHC version then there would be Homebrew linking problem that wrapper is present in several formulae.

Reply:
We can `rm` the wrapper built in the other formulae. You're going to be `rm`-ing them anyway, right?

Reply:
So what you suggest is that `HLS@ghc8.6` and `8.8` had the `depends_on "haskell-language-server"`, right?
I think I prefer that each `hls` depended on `hls-wrapper`... could you live with that? ;]

Reply:
> So what you suggest is that `HLS@ghc8.6` and `8.8` had the `depends_on "haskell-language-server"`, right?

Yes.

> I think I prefer that each `hls` depended on `hls-wrapper`... could you live with that? ;]

Not really, no. 

They build the same things from the same source and require manual intervention to come out the way we want them to. There's also the added maintenance burden of keeping the HLS and HLS-wrapper formulae versions in sync, when it wouldn't be needed if they were just one formula.

In fact, I'd much rather just have a single `haskell-language-server` formula that works for all the versions rather than separate ones for each ghc version.

Reply:
Hmm, ok. I will try to make this into a single formula. Why didn't I think of that...? 🤦